Translation/Importer/FileImporter.php

```diff
@@ -76,6 +79,12 @@ public function import(\Symfony\Component\Finder\SplFileInfo $file, $forceUpdate
                 }
                 $transUnit = $this->storage->getTransUnitByKeyAndDomain($key, $domain);
 
+                if(isset($repeatedControl[strtolower($key)])){
+                    print sprintf("\n\t The key '%s' is already in this file, need to be purged!, Ignoring second appearance.\n", $key);
+                    continue;
+                }
+                $repeatedControl[strtolower($key)] = true;
```

Comment:
Hi, the error discribed in https://github.com/lexik/LexikTranslationBundle/issues/120 is also related to your database configuration, I mean if it's case sensitive or not. Your code will fix the error for a case insensitive database, but it will miss some keys for a case sensitive database.

And don't use any `print` or `echo` statements, a class must not display anything by using this kind of statements.